.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
+      - name: Lint
```

Comment:
I'd suggest you move the lint step to after the Install Dependencies step.

Reply:
+1

Reply:
Opps my bad! 😅